features/step_definitions/user_steps.rb

```diff
@@ -13,14 +13,29 @@ def see_journey_picker?
   @see_journey_picker
 end
 
+def page_mapping(page_name)
+  case page_name
+  when 'Verify start' then
+    'start'
+  when 'IDP picker' then
```

Comment:
I would argue the "IDP picker page" is `/choose-a-certified-company` not `sign-in`

Reply:
@jakubmiarka What would you suggest the "sign-in" page to be called? It was called the "IDP picker page" because it gives the user a selection of IDP's to pick from. I would call the "choose-a-certified-company" page "IDP choices page"

Reply:
I would call it just a sign-in page (also the url is `/sign-in`). "IDP picker" is a standard convention used by designers and user researchers to describe the `/choose-a-certified-company`... It will be then consistent with "Country picker" where the url is `/choose-a-country` and actually the page has a similar layout too.

Reply:
I have changed this to "IDP sign in" as it was quite ambiguous just having it as "sign in".